### ISSUE TITLE
SimpleGui and PyQt4 - no attribute 'setupViewport'

### ISSUE THREAD

**Issue by tpaviot** [tpaviot/pythonocc-core#376]
When resizing the window:
```
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OCC\Display\qtDisplay.py", line 215, in re
sizeGL
    self.setupViewport(width, height)
AttributeError: 'qtViewer3d' object has no attribute 'setupViewport'
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OCC\Display\qtDisplay.py", line 215, in re
sizeGL
    self.setupViewport(width, height)
AttributeError: 'qtViewer3d' object has no attribute 'setupViewport'
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OCC\Display\qtDisplay.py", line 215, in re
sizeGL
    self.setupViewport(width, height)
AttributeError: 'qtViewer3d' object has no attribute 'setupViewport'
```

**Comment by jf---** [comment]
hi @tpaviot  this is addressed in PR #375

**Comment by tpaviot** [comment]
Fixed by commit 2a9210f
